Description: Support armhf, kfreebsd-* and s390x arch
Author: Benjamin Drung <bdrung@debian.org>
Bug-Debian: http://bugs.debian.org/644630

--- a/apt-mirror
+++ b/apt-mirror
@@ -263,7 +263,7 @@
 	next;
     }
 
-    if($config_line =~ /deb-(alpha|amd64|armel|arm|hppa|hurd-i386|i386|ia64|lpia|m68k|mipsel|mips|powerpc|s390|sh|sparc)/) {
+    if($config_line =~ /deb-(alpha|amd64|armel|arm|armhf|hppa|hurd-i386|i386|ia64|kfreebsd-i386|kfreebsd-amd64|lpia|m68k|mipsel|mips|powerpc|s390|s390x|sh|sparc)/) {
 	push @config_binaries, [$1, @config_line];
 	next;
     }
