Description: Add a shebang line to clean.sh
Author: Benjamin Drung <bdrung@ubuntu.com>
Bug-Ubuntu: https://launchpad.net/bugs/649364

diff --git a/apt-mirror b/apt-mirror
--- a/apt-mirror
+++ b/apt-mirror
@@ -625,6 +625,7 @@ if(get_variable("_autoclean")) {
     print "$size_output in $total files and " . scalar(@rm_dirs) . " directories can be freed.\n";
     print "Run " . get_variable("cleanscript") . " for this purpose.\n\n";
 
+    print CLEAN "#!/bin/sh\n\n";
     print CLEAN "cd " . get_variable("mirror_path") . " || exit 1\n\n";
     print CLEAN "echo 'Removing $total unnecessary files [$unnecessary_bytes bytes]...'\n";
     foreach (@rm_files) {
